tracim/tracim/lib/webdav/sql_resources.py

```diff
@@ -970,6 +970,7 @@ def move_file(self, destpath):
         parent = self.content.parent
 
         with new_revision(self.content):
+            # Rename
```

Comment:
Comment must be formatted like this: 
# GM - 2018-03-09 - INFO - Case of renaming the file